### ISSUE TITLE
The game crashes after starting - Critical Ops

### ISSUE THREAD

**Issue by Danl54** [Perfare/Zygisk-Il2CppDumper#145]
I tried with Among Us and it woks but for the game:  **Critical  Ops**. It does just crash without creating a `dump.cs`. I don't know what is the problem with it but I hope you can help me. Seems like the metadata is encrypted or sth.

https://github.com/Perfare/Zygisk-Il2CppDumper/assets/114314300/a26350c5-a823-42bb-8d9b-d50d0f2acd66

**Comment by ADAMOUMOU** [comment]
same problem with critical ops

**Comment by Danl54** [comment]
> same problem with critical ops

yep they enc their metadata...

**Comment by ADAMOUMOU** [comment]
And is there a way to decrypt thème or something like that

**Comment by ADAMOUMOU** [comment]
Or it can be a hooking detection or some shits like that

**Comment by Danl54** [comment]
> Or it can be a hooking detection or some shits like that

no idea to be honest bro

**Comment by ADAMOUMOU** [comment]
Have you a discord ?

**Comment by Danl54** [comment]
yes ofc Dani54#8888

**Comment by Donkeycon123** [comment]
Have u fixed this problem?

**Comment by Danl54** [comment]
> Have u fixed this problem?

No i haven't. The dumper needs  to be updated in order to fix it.

**Comment by ADAMOUMOU** [comment]
sorry i didn't see your message with your discord i add you now

**Comment by gixousiyq** [comment]
Did you fix this problem?

**Comment by ADAMOUMOU** [comment]
> Did you fix this problem?

No i didn't fixed it but i think you can try with the Frida toolkit: https://frida.re

**Comment by gixousiyq** [comment]
What does(Frida toolkit) do?

**Comment by ADAMOUMOU** [comment]
It helps you to debug programs you want on android and ios bit i will learn so i know as much as you. I think you need to have some bases in reverse engineering and arm/arm64 assembly

**Comment by gixousiyq** [comment]
Ok thanks I will search more on it and see
Thank you for your respone!

**Comment by Danl54** [comment]
> It helps you to debug programs you want on android and ios bit i will learn so i know as much as you. I think you need to have some bases in reverse engineering and arm/arm64 assembly

Dude what is your discord. You can contact me and we can do some tests with frida together. I will try learning how it works too.

**Comment by gixousiyq** [comment]
As what I've seen, This frida toolkits can hook functions and you can make mod menus with it. But I don't know if it can extract offsets from the game, I still need to do more search on it.

**Comment by Danl54** [comment]
> As what I've seen, This frida toolkits can hook functions and you can make mod menus with it. But I don't know if it can extract offsets from the game, I still need to do more search on it.

yeah I am just taking a look at it and downloading it

**Comment by ADAMOUMOU** [comment]
Hey,
Have you digged more about Frida ?
Here is my discord so we can talk if you want: adamouchgobrr

**Comment by ADAMOUMOU** [comment]
Here is some interesting ressources if you want: https://github.com/kylesmile1103/Learn-Frida

**Comment by Danl54** [comment]
> Here is some interesting ressources if you want: https://github.com/kylesmile1103/Learn-Frida

Yeah I'm adding you so we can discuss more on it later. I haven't digged too much into it.
